tests/spread/testcraft/grammar/for/testcraft.yaml

```diff
@@ -0,0 +1,46 @@
+name: grammar
+base: ubuntu@24.04
+version: "1.0"
+summary: test
+description: |
+  Exercise testcraft's advanced grammar statement `for`.
+
+platforms:
+  platform1:
+    build-on: amd64
+    build-for: riscv64
+  platform2:
+    build-on: amd64
+    build-for: s390x
+
+parts:
+  hello-world:
+    plugin: dump
+
+    # grammar-only string
+    source:
+    - for platform1: for-platform1
+    - for platform2: for-platform2
```

Comment:
Has this become correct indentation now? I thought we were pushing against having sequences be on the same level as their parent.